example/lib/main.dart

```diff
@@ -14,29 +15,24 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  String _platformVersion = 'Unknown';
+  String _platformVersion = ExampleStrings.platformVersion;
 
   @override
   void initState() {
     super.initState();
-    initPlatformState();
+    //initPlatformState();
```

Comment:
if you don't need this line, please remove it

Reply:
Removed